test/controllers/api_users/permissions_controller_test.rb

```diff
@@ -240,49 +244,69 @@ class ApiUsers::PermissionsControllerTest < ActionController::TestCase
 
       stub_policy current_user, api_user, update?: false
```

Comment:
Looks like we stub the policy in some places but not others. I wonder if it would be best to use either a generic user + stubbed policy (more in line with the other specs), or a superadmin without a stubbed policy throughout. Minor though!

Reply:
I've added stubbing but left it with the superadmin to keep the tests reading more "realistic" as a non superadmin/admin wouldn't be able to do this. We can maybe tidy this up in future if we make these controller tests more integration style and/or vice-versa.